resources/scss/Findologic.scss

```diff
@@ -0,0 +1 @@
+@import '../../vendor/plentymarkets/plugin-ceres/resources/scss/variables';
```

Comment:
* Rename this file to `findologic.scss`.
* Import also `resources/scss/views/ItemList/Components/Filter/item-filter-list.scss` and change to gulp task so it only compiles `findologic.scss`.